apps/etterlatte-saksbehandling-ui/client/src/components/behandling/attestering/oppsummering/oversikt.tsx

```diff
@@ -61,7 +61,9 @@ export const Oversikt = ({ behandlingsInfo }: { behandlingsInfo: IBehandlingInfo
   }
 
   const fattetDato = behandlingsInfo.datoFattet
-    ? formaterStringDato(behandlingsInfo.datoFattet) + ' kl. ' + formaterStringTidspunkt(behandlingsInfo.datoFattet)
+    ? formaterStringDato(behandlingsInfo.datoFattet) +
+      ' kl. ' +
+      formaterStringTidspunktTimeMinutter(behandlingsInfo.datoFattet)
```

Comment:
Hvorfor bruker du ikke `formaterDatoMedTidspunkt()`? 

Den gjør akkurat dette (minus "kl."), men at det er klokkeslett sier seg litt selv når man ser `07.02.2024 12:33` 🤷‍♂️

`export const formaterDatoMedTidspunkt = (dato: Date) => format(new Date(dato), 'dd.MM.yyyy HH:mm').toString()`

Reply:
For å slippe å gå opp den løypa med sbere om hvorfor de vil ha "kl" foran eller ei, mao holde det helt likt som det var rent funksjonelt.

Reply:
Du kunne i det minste brukt date-fns sin formateringsfunksjon for å unngå de greiene her: 

```
formaterStringDato(behandlingsInfo.datoFattet) +
      ' kl. ' +
      formaterStringTidspunktTimeMinutter(behandlingsInfo.datoFattet)
```

https://date-fns.org/docs/format#examples